lambdas/process-cri-callback/src/main/java/uk/gov/di/ipv/core/processcricallback/service/CriCheckingService.java

```diff
@@ -231,11 +231,11 @@ public JourneyResponse checkVcResponse(
 
             // Check CI levels against the lowest confidence identity requested so we don't send the
             // user on an unnecessary mitigation journey.
-            var lowestConfidenceRequested =
-                    clientOAuthSessionItem.getLowestStrengthRequestedGpg45Vot(configService);
+            var targetVot = clientOAuthSessionItem.getTargetVot();
             var journeyResponse =
-                    ciMitUtilityService.getMitigationJourneyIfBreaching(
-                            cis, lowestConfidenceRequested);
+                    ciMitUtilityService.getMitigationJourneyIfBreaching(cis, targetVot);
```

Comment:
Don't need inherited identity config bc update to gpg45 if not achieved in CheckExistingIdentity